data/crac-file/crac-io-json/src/test/java/com/farao_community/farao/data/crac_io_json/RangeActionImportExportTest.java

```diff
@@ -0,0 +1,291 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.data.crac_io_json;
+
+import com.farao_community.farao.data.crac_api.NetworkElement;
+import com.farao_community.farao.data.crac_api.RangeDefinition;
+import com.farao_community.farao.data.crac_impl.AbstractRemedialAction;
+import com.farao_community.farao.data.crac_impl.range_domain.Range;
+import com.farao_community.farao.data.crac_impl.range_domain.RangeType;
+import com.farao_community.farao.data.crac_impl.remedial_action.range_action.*;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static com.farao_community.farao.data.crac_io_json.RoundTripUtil.roundTrip;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+public class RangeActionImportExportTest {
+    @Test
+    public void pstRangeJsonCreator() {
+        PstRange pstRange = new PstRange(
+            "pstRangeId",
+            "pstRangeName",
+            "RTE",
+            new ArrayList<>(),
+            new ArrayList<>(),
```

Comment:
Same, don't you want a constructor without UsageRule nor Range?

Reply:
alright